### PR TITLE
Design a clone webpage of Apple

### PR DESCRIPTION
**Pull request by kagaramag**
## Title

Design a clone webpage of Apple

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
\ No newline at end of file
+# HTML-Forms
+This project consists of cloning the Apple webpage using HTML and CSS.
+
+# Built With
+- HTML
+- CSS
+
+Link to live page
+https://raw.githack.com/djallas/magpies-apple-clone/master/index.html
```
Comment (MauricioRobayo) [inline README.md]: Please double-check this link, it is not working. You should paste the URL to the `index.html` of your desired branch into the https://raw.githack.com/ input. The URL you should paste there for this project is https://github.com/djallas/magpies-apple-clone/blob/develop/index.html, then you can share the URL given under `Use this URL for development`: https://raw.githack.com/djallas/magpies-apple-clone/develop/index.html.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+  width: 88%;
+  padding: 0 1%;
+  float: left;
+}
+
+.welcome-section {
+  height: 80vh;
+  background-image: url('../images/navigator.jpg');
+  background-attachment: scroll;
+  background-size: initial;
```
Comment (MauricioRobayo) [inline css/style.css]: Change the `background-size` from `initial` to `cover` to make it always fill the whole screen.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+              Everyone has something to share. See how people are using iPad in
+              their unique ways to add to the world’s story.
+            </h2>
+            <h3>Watch the “Your Verse” film and explore the stories</h3>
+          </div>
+        </div>
+      </section>
+      <section class="promo-ads">
+        <div class="row">
+          <div class="col-25">
```
Comment (MauricioRobayo) [inline index.html:60]: Each of this four articles should be a link. Use the `a` tag to wrap the content and make it clickable.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi @djallas,

Great job so far, you have done very well and the project is coming into place. Very nice use of semantic HTML. Please work on the following issues:

* [ ] Enable Stickler on this repository, take a look at [these instructions](https://github.com/microverseinc/linters-config/tree/master/css).
* [ ] Tha background image for the main section is repeating itself instead of covering the whole background.
* [ ] The input for the search on the navbar is overflowing the navbar.
* [ ] The four articles at the bottom should be links (to other articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))
* [ ] Try to make the footer of your page resemble the original one. It should look very similar to this:
image

Take a look at this screenshot from your project for the search box on the navbar, the background image, and the footer from your page:
image

Status: **Request changes**.

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @djallas     
#### Great Job so far, but you haven't implemented some of the required changes from the last review.
#### General 
- [ ] The search on the navbar is still overflowing.


 - [ ] The four articles at the bottom should be links 


- [ ] Try to make the footer of your page resemble the original one. It should look very similar to this:
### Make changes and submit another code review request.
*Happy coding*

**Comment by kagaramag** [comment]
> ## Hi @djallas 
> #### Great Job so far, but you haven't implemented some of the required changes from the last review.
> #### General
> * [ ]  The search on the navbar is still overflowing.
>   
> * [ ]  The four articles at the bottom should be links
>   
> * [ ]  Try to make the footer of your page resemble the original one. It should look very similar to this:
> 
> ### Make changes and submit another code review request.
> _Happy coding_

Hello @kobiyoyo 
Thank you for your feedback. However, I am not sure if the codes you reviewed are showing the same result as the last commit I had. for instance:

1. For search bar, I applied styles to make it not overfloating against available width
```
.grab-search {
  max-width: 98%;
  float: left;
  margin: 0 1%;
  border: 1px solid #383838 !important;
  background: rgba(0, 0, 0, 0.5);
  border-radius: 10px !important;
}
```

2. The four articles at the bottom have links, beliw there are codes for one article
```
<div class="col-25">
 <div class="image">
  <a href="http://www.apple.com/iphone-5s/">
   <img src="images/promo_iphone5s_2x.jpg" alt="iPhone5s" />
   </a>
  </div>
</div>
```

3. I want to confirm if the footer is looking as a screenshot attached below
image

@kobiyoyo + TSEs Can you check for me if everything is okay, and let me know if there is a recommendation/suggestion?
Note:
- _The recent commit, I just changed **font-weight** and **letter-spacing** in the footer content_
- I know that I changed a few things after the last feedback, it is just to ask for clarification in this comment.

Thanks

**Review by rOluochKe — APPROVED** [review]
Hello @djallas 

### You have done well, therefore, I approve of this project. Congratulations :clap: :boom:

### Now merge and submit a project completion form from your dashboard. I wish you the best in the next project. 
- Happy coding!

**Pull request merged**